### ISSUE TITLE
Add openvpn access groups resource

### ISSUE THREAD

**Issue by iamunni** [OpenVPN/terraform-provider-openvpn-cloud#63]
resource - https://openvpn.net/cloud-docs/developer/openvpn-api.html#UUID-c0946de0-43ab-347e-b5db-4836cb02dea9
api is available

**Comment by iamunni** [comment]
@vladimir-kozyrev @michaelfmnk can I work on this?

**Comment by iamunni** [comment]
there is an open issue, which is for service and accessgroups - #14 
I see that the changes for service is already being added. hence raised a new issue for just accesgroups. let me know if I should close this and continue the discussion on #14

**Comment by vladimir-kozyrev** [comment]
@iamunni feel free to work on this. :)
There are no objections from me.

**Comment by iamunni** [comment]
@vladimir-kozyrev awesome. can you assign this issue to me

**Comment by iamunni** [comment]
there is some issue with the GET /api/beta/access-groups api . I'm getting the following error from yest. Tried with a new account, issue started showing up after working fine for couple of times
![image](https://github.com/OpenVPN/terraform-provider-openvpn-cloud/assets/62323890/2e09d054-75ad-4eb3-9d80-1b8c9a8d044d)
![image](https://github.com/OpenVPN/terraform-provider-openvpn-cloud/assets/62323890/5aa596a5-eeed-4c86-8a5c-39385bcac712)

have send a mail to support@openvpn.com, so far no response. waiting for updates from them

**Comment by iamunni** [comment]
@dsommers would you be able to help me out here

**Comment by iamunni** [comment]
no reply from openvpn team

**Comment by dlenart-openvpn** [comment]
Hello there! Apologies for late response. We have been working on this at the side of OpenVPN, and I think we managed to prepare a fix. Now, we are working to do the release. I will update you early next week where we are.

**Comment by michaelfmnk** [comment]
Hey! Here is the [PR#60](https://github.com/OpenVPN/terraform-provider-openvpn-cloud/pull/60) with added usergroups resource. It would be great if you could take a look and see if it has everything you need.

**Comment by iamunni** [comment]
@michaelfmnk [PR#60](https://github.com/OpenVPN/terraform-provider-openvpn-cloud/pull/60) is for [user-groups](https://openvpn.net/cloud-docs/developer/openvpn-api.html#UUID-eed5e88f-213d-7bdd-01a3-a683b70ba1e0). this issue is for [access-groups](https://openvpn.net/cloud-docs/developer/openvpn-api.html#UUID-c0946de0-43ab-347e-b5db-4836cb02dea9)

**Comment by iamunni** [comment]
@dlenart-openvpn awesome

**Comment by unnirapido** [comment]
@dlenart-openvpn hey is the fix released?

**Comment by dlenart-openvpn** [comment]
@unnirapido hello there! Not really. We have been lately working on user groups support, and did not manage to prepare access groups within same release as well. But we continue work on this, and will let you know once this is ready. Should not take more than 2-3 weeks.

**Comment by unnirapido** [comment]
@dlenart-openvpn is the fix released now?

**Comment by sahaqaa** [comment]
We've published new version of Terraform Provider to Registry: https://registry.terraform.io/providers/OpenVPN/cloudconnexa/latest

Going forward only https://registry.terraform.io/providers/OpenVPN/cloudconnexa/latest will be developed.

Please give new provider a try, hopefully it will fix your issue.

In case if problem persists - please open new "Issue" (https://github.com/OpenVPN/terraform-provider-cloudconnexa/issues)

Sorry for inconvenience.
